tests/src/Mod/Part/App/PartFeature.cpp

```diff
@@ -60,3 +60,72 @@ TEST_F(FeaturePartTest, testGetElementName)
 #endif
     // TBD
 }
+
+TEST_F(FeaturePartTest, create)
+{
+    // Arrange
+
+    // A shape that will be passed to the various calls of Feature::create
+    auto shape {TopoShape(BRepBuilderAPI_MakeVertex(gp_Pnt(1.0, 1.0, 1.0)).Vertex(), 1)};
+
```

Comment:
A note:  in this context, _boxes[0] to _boxes[6] and or _common are already available as TopoShapes, and it might be useful to define a helper object for a simple vertex if we'll reuse it a lot in testing.

No need to make a change to this test, as it is fine.

Reply:
Yes, I noticed that there already were shapes inside the document but preferred to clean it and re-populate with shapes explicitly  created to have more control on the test.

It's more likely that I'll use the pre-created shapes in more complex tests.

Thanks anyway for the tip!